algorithms/symmetry/__init__.py

```diff
@@ -349,7 +349,6 @@ def resolution_filter_from_array(intensities, min_i_mean_over_sigma_mean, min_cc
     """Run the resolution filter using miller array data format."""
     rparams = phil_defaults.extract().resolutionizer
     rparams.nbins = 20
-    rparams.plot = False
     resolutionizer = Resolutionizer(intensities, rparams)
```

Comment:
I note in some of the `command_line` code below we have `r = Resolutionizer()` - which could help readability esp. if we chose to go down the `r.ENUM` route